app/server.go

```diff
@@ -9,9 +9,11 @@ import (
 	"github.com/zillow/howwegoatzillow/libs/server"
 )
 
+type MyServer struct{ *server.Server }
+
 // This is a very crude representation of what each application looks like.
 // Everything from here and underneath is this application domain and should be well tested.
-func NewServer(service MyService) *server.Server {
+func NewMyServer(server *server.Server, service *MyService) *MyServer {
```

Comment:
New type since there is already a NewServer in the common providers which returns the base server. And you cannot have two providers for the same type. So creating a MyServer which extends the base server with your application specific routes.